src/common/MovieTile/index.js

```diff
@@ -1,6 +1,6 @@
 import { Tile, Poster, Content, Title, Year, Tags, Tag, TagName, Rating, Star, Rate, Votes } from "./styled";
 import { img } from "../../core/apiCodes";
-import { Genres } from "../../features/MoviesList/Genres";
+import { Genres } from "../../features/movies/MoviesList/Genres";
```

Comment:
Maybe it will be a good idea to have Genres folder outside MoviesList folder since we need it not only in movie list page but also in Movie details etc?

Reply:
Good idea, I'll move Genres off the movies list :)